### ISSUE TITLE
Untranslated h5 headings in "Edit Collection > Assign Roles" after page reload

### ISSUE THREAD

**Issue by saschaszott** [DSpace/dspace-angular#1727]
## Describe the bug

The content of all h5 elements in Edit Collection > Assign Roles is not translation if you switch the UI language.

### Screenshot (before language switch):

![image](https://user-images.githubusercontent.com/952735/179241171-7ea8595f-efb8-47ff-8241-7d74089749b4.png)

### Screenshot (after language switch):

![image](https://user-images.githubusercontent.com/952735/179241088-d56d88bd-432c-4202-a030-b9820958e06e.png)

## To Reproduce

1. Open "Edit Collection"
1. Open "Assign Roles" tab
1. Change language

## Expected behavior

Translation mechanism should work properly after changing language of UI.

**Comment by tdonohue** [comment]
@saschaszott : What language are you changing to in these screenshots?  (It's unclear to me.)  Each language must define the same keys in order for them to be properly translated in the UI.  So, in the screenshots you shared above, this implies to me that one of the language files is *missing* those keys -- but, it'd be difficult for us to track down without more information.

As I noted in the other ticket you created, we also welcome small PRs to fix these issues if you have to do so.  All translations are done by volunteers (we have no way to auto-translate the UI), so we are dependent on multilingual volunteers helping us to translate our UI into those other languages.

**Comment by saschaszott** [comment]
You can switch from English to English - the error does not seem to have any connection to selected language. Is more like a page reload issue (you even do not need to change the language at all - go to http://your-ds-instance/collections/<id>/edit/roles and reload the page).

**Comment by tdonohue** [comment]
@artlowel : wanted to alert you to this as it almost sounds like a caching/reload issue here (and I know you and your team have been working on page reload / initial load fixes in the past.  

In any case, I'll move this to the 7.4 board if anyone wants to claim it.

**Comment by tdonohue** [comment]
@saschaszott :  I was just trying to reproduce this and I've realized I cannot reproduce it on https://demo7.dspace.org/  Here's what I've tried doing:
1. Login as an Admin
2. Visit a Collection's "Assign Roles" tab, e.g. https://demo7.dspace.org/collections/07186f80-e22b-4a48-9697-3931dbe3689a/edit/roles
3. Reload page.  No change in display
4. Change language to German.  Language changes... H5 headers are still there.  They briefly remain in English though, and then the page auto-reloads and they appear in German.
5. Change language back to English.  Language changes back.  H5 headers are still there.  They briefly remain in German though and then the page auto-reloads and they appear in English.

So, I do see that the language change behavior causes these headers not to reload as quickly as the rest of the text on the page.  But, I never see them appear as i18n keys (like in your second screenshot in the description).
